include/zephyr/posix/unistd.h

```diff
@@ -55,6 +55,153 @@ pid_t getpid(void);
 unsigned sleep(unsigned int seconds);
 int usleep(useconds_t useconds);
 
+#ifdef CONFIG_POSIX_SYSCONF
+#define SUPPORTED_POSIX1_VERSION 200809L
+
+#define __z_posix_sysconf_SC_AIO_LISTIO_MAX		  (-1L)
+#define __z_posix_sysconf_SC_ARG_MAX			  (-1L)
+#define __z_posix_sysconf_SC_AIO_MAX			  (-1L)
+#define __z_posix_sysconf_SC_AIO_PRIO_DELTA_MAX		  (-1L)
+#define __z_posix_sysconf_SC_ATEXIT_MAX			  (-1L)
+#define __z_posix_sysconf_SC_BC_BASE_MAX		  (-1L)
+#define __z_posix_sysconf_SC_BC_DIM_MAX			  (-1L)
+#define __z_posix_sysconf_SC_BC_SCALE_MAX		  (-1L)
+#define __z_posix_sysconf_SC_BC_STRING_MAX		  (-1L)
+#define __z_posix_sysconf_SC_CHILD_MAX			  (-1L)
+#define __z_posix_sysconf_SC_CLK_TCK			  CONFIG_SYS_CLOCK_TICKS_PER_SEC
```

Comment:
IIRC, this should be 1M on any POSIX system, regardless of what the tick rate is (I may be misremembering)

Reply:
I'll double check that, I based values of variables on the https://github.com/zephyrproject-rtos/zephyr/pull/60030 and found some that doesn't  have correct value assigned. Do you think that https://docs.zephyrproject.org/latest/services/portability/posix/conformance/index.html# is a good source of information or it's better to stick to open group specification?

Reply:
After some digging I found that's clock ticks per second. Often it's set to 100.  Someone said:
> it's the number of times the timer interrupts the CPU for scheduling and other tasks

https://stackoverflow.com/questions/19919881/sysconf-sc-clk-tck-what-does-it-return#comment29641173_19919970

Reply:
I'm not sure if this one has been resolved yet. I'm ok with it going in as-is though.

Reply:
I decided to go with `100L` as it's a commonly used value for the symbol 
https://github.com/zephyrproject-rtos/zephyr/blob/1bc840c208006d1df7bf5e9bc6416d767222dcb5/include/zephyr/posix/unistd.h#L356